app.Makefile

```diff
@@ -49,10 +50,13 @@ app/build:: ;
 app/install:: app/build \
               .build/var/MARKETPLACE_TOOLS_PATH \
               .build/var/APP_DEPLOYER_IMAGE \
-              .build/var/APP_PARAMETERS
-	$(MARKETPLACE_TOOLS_PATH)/scripts/start.sh \
+              .build/var/APP_PARAMETERS \
+              .build/marketplace/dev
```

Comment:
nit: sort a-z and put this before var one?

Reply:
Done.